README.md

```diff
@@ -144,11 +144,13 @@ A preference is what will allow IAM to identify users for targeting and segmenta
 3.  AccessToken - This is the token provided by the internal RAuthentication SDK as the "accessToken" value
 
 To help IAM identify users, please set a new preference every time a user changes their login state i.e. when they log in or log out.  
-Not all identifiers have to be provided.
+During log out please call  `registerPreference()` with nil parameter instead of  `IAMPreference` object with nil properties.  
```

Comment:
I don't think we need "instead of  `IAMPreference` object with nil properties".

Should it be "After logout is complete" rather than "During logout" ?

Reply:
Right "during logout" can be difficult :D I wanter do be sure that no one thinks of an idea to pass an empty object instead of nil but I think you are right that it's covered by previous part of the sentence